privacyidea/lib/tokens/yubikeytoken.py

```diff
@@ -298,7 +298,11 @@ def check_otp(self, anOtpVal, counter=None, window=None, options=None):
 
         uid = msg_hex[0:12]
         log.debug("uid: {0!r}".format(uid))
-        log.debug("prefix: {0!r}".format(binascii.hexlify(modhex_decode(yubi_prefix))))
+        try:
+            log.debug("prefix: {0!r}".format(binascii.hexlify(modhex_decode(yubi_prefix))))
```

Comment:
This is just a debug output, can't we just print the prefix as is instead of converting it to a hex-string?
The check if the prefix is valid should be done separately from the debug message.

Reply:
Sounds sensible to me.

Will do.

Reply:
Done in 0bba2428